plugins/cake/cake_

```diff
@@ -0,0 +1,131 @@
+#!/bin/sh
+# -*- sh -*-
+
+: << =cut
+
+=pod
+
+=encoding UTF-8
+
+=head1 NAME
+
+cake_ - Plugin to monitor cake's backlog, dropped, overlimits and requeues
+
+=head1 CONFIGURATION
+
+None needed.
+
+=head1 INTERPRETATION
+
+Cake, also known as sch_cake is a modern bandwidth limiter, which eliminates
+buffer bloat over slow links. It's also capable to give flows, hosts and each 
+flow of each host a fair part of the avaible bandwidth.
+
+This plugin allows for a monitor of the pressure on the qdisc, by monitoring key
+values.
+
+=head1 SEE ALSO
+
+Take a look at "man cake" to get more information about cake.
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf suggest
+
+=head1 AUTHORS
+
+RubenKelevra <ruben@vfn-nrw.de>
+
+work based on the tc plugin, authors:
+Steve Schnepp <steve.schnepp@gmail.com>,
+Samuel Smith <esaym@cpan.org>,
+Nye Liu <nyet@nyet.org>
+
+=head1 LICENSE
+
+GPLv2 or later
+
+=cut
+
+DEVICE=${0##*/cake_}
+
+tc_cake_sent() {
+	/sbin/tc -s qdisc show dev "$1" | grep -E "^ Sent" | tr ',' ' ' | tr ')' ' ' 
+}
+tc_cake_backlog() {
+	/sbin/tc -s qdisc show dev "$1" | grep -E "^ backlog" | tr 'p' ' ' 
+}
+
+case "$1" in
+    autoconf)
+	if [ -r /proc/net/dev ]; then
+	    echo yes
+	    exit 0
+	else
+	    echo "no (/proc/net/dev not found)"
+	    exit 1
+	fi
+	;;
+    suggest)
```

Comment:
Just as a thought: personally I prefer multigraph plugins due to their ease of configuration (automatically following changes with regard to obsolete/new interfaces).
Maybe you want to turn this plugin into a multigraph plugin instead of a suggest-style plugin?
(just a matter of taste - the decision is up to you)